packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -327,13 +334,33 @@ abstract class BindingBase {
     registerServiceExtension(
       name: name,
       callback: (Map<String, String> parameters) async {
-        if (parameters.containsKey(name))
-          await setter(double.parse(parameters[name]));
+        if (parameters.containsKey(name)) {
+          final double value = double.parse(parameters[name]);
+          await setter(value);
+          _postExtensionToggledEvent(name, value);
+        }
         return <String, dynamic>{ name: (await getter()).toString() };
       }
     );
   }
 
+  void _postExtensionToggledEvent(String name, dynamic value) {
```

Comment:
Nit:
Toggled --> Changed

toggled implies bool behavior but generally this is about any time one of these extensions changed.